cspell.config.yaml

```diff
@@ -3,8 +3,6 @@ language: en-US
 useGitignore: true
```

Comment:
We should specify `usePnP: true` since we are using Plug'n'Play, and hopefully that is all that is needed for CSpell to access to our Yarn dependencies.